sdk/translation/azure-ai-translation-document/src/main/java/com/azure/ai/translation/document/models/Status.java

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.translation.document.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/**
+ * List of possible statuses for job or document.
+ */
+public final class Status extends ExpandableStringEnum<Status> {
```

Comment:
The name `Status` is too generic. We should call this `DocumentTranslationStatus` or something more specific.

Reply:
This is something that came up during Dotnet SDK review too and we have noted this and will take care of it before GA release.